.github/workflows/helm-lint.yaml

```diff
@@ -0,0 +1,41 @@
+name: Reusable workflow for linting helm charts
+
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: "The ref name to checkout the repository."
+        required: false
+        default: ${{ github.event.repository.default_branch }}
+        type: string
+      lint-config-path:
+        description: "The path to the lint configuration file (For an example see https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml)."
+        required: false
+        default: ".github/lint-config.yaml"
+        type: string
+      helm-version:
+        description: "The helm version for setting up poetry."
+        required: false
+        default: "v3.4.0"
+        type: string
+
+jobs:
+  helm-lint:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ inputs.ref }}
+          fetch-depth: 2
```

Comment:
why a fetch-depth of 2?

Reply:
The lint action checks helm chart changes before linting. For that it compares the current helm chart with the helm chart in the previous commit

Reply:
Okay good to know maybe add this as a comment in the file